Source/Details/ASDataController.mm

```diff
@@ -437,6 +437,29 @@ - (void)waitUntilAllUpdatesAreCommitted
   [self _scheduleBlockOnMainSerialQueue:^{ }];
 }
 
+- (BOOL)isProcessingUpdates
+{
+  ASDisplayNodeAssertMainThread();
+  long waitResult = dispatch_group_wait(_editingTransactionGroup, DISPATCH_TIME_NOW);
+  return (waitResult == 0 ? NO : YES);
```

Comment:
Are you maybe thinking about `layoutIfNeeded`? That would absolutely make sense 👍